src/components/Sidebar/ChoiceNode.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { FC, ChangeEvent } from 'react';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faChevronRight, faEdit } from '@fortawesome/free-solid-svg-icons';
+import { usePathwayContext } from 'components/PathwayProvider';
+import styles from './Sidebar.module.scss';
+import { FormControl, makeStyles, TextField, InputAdornment, Input } from '@material-ui/core';
+
+interface ChoiceNodeProps {
+  transition: string;
+}
+
+const ChoiceNode: FC<ChoiceNodeProps> = ({ transition }) => {
+  const { pathway } = usePathwayContext();
+
+  const useStyles = makeStyles(theme => ({
+    formControl: {
+      margin: theme.spacing(1, 0),
+      minWidth: 120,
+      display: 'flex',
+      flexWrap: 'wrap'
+    },
+    rename: {
+      fontSize: '2rem',
+      display: 'flex',
+      flexWrap: 'wrap'
+    }
+  }));
+
+  const classes = useStyles();
+  const initial = pathway !== null ? pathway.states[transition].label : 'Choice Node';
+
+  // TODO: in PATHWAYS-256 use the data from the pathway
+  const [choice, setChoice] = React.useState('');
+  const [label, setLabel] = React.useState<string>(initial);
+
+  const handleChoiceChange = (event: ChangeEvent<{ value: unknown }>): void => {
+    // TODO: in PATHWAYS-256 need to update the state of the node
+    setChoice(event.target.value as string);
+  };
+
+  const handleLabelChange = (event: ChangeEvent<{ value: unknown }>): void => {
+    // TODO: in PATHWAYS-256 need to update the transition of the previous node
+    // and the state of the choice node
+    setLabel(event.target.value as string);
+  };
+
+  // TODO: in PATHWAYS-256 the forward button needs to select the newly created choice node
+  return (
+    <div>
+      <div className={styles.choiceNode}>
+        <FormControl className={classes.formControl}>
+          <Input
```

Comment:
This is a design question for @noranda should the name be editable by simply clicking on it? Or should the user need to click the edit icon first?

Reply:
The entire form element should be clickable to edit. The edit icon is a visual indicator that the field is editable, so it's more of an [input adornment](https://material-ui.com/components/text-fields/#input-adornments).